andela/storm-backend
# #168093149 add template folder to utils folder

#### What this PR does
This PR will add templates folder to utils folder

#### Tasks completed
- Add templates folder to utils folder
- Move verifyEmailMessage file to the templates folder

#### How this can be manually tested
- Clone this branch
- Change directory into it
- Check the utils folder for the templates folder

#### What are the relevant pivotal tracker stories?
[#168093149](https://www.pivotaltracker.com/story/show/168093149)
